SwifterSwift.xcodeproj/project.pbxproj

```diff
@@ -1125,6 +1169,7 @@
 			attributes = {
 				LastSwiftUpdateCheck = 0900;
 				LastUpgradeCheck = 0930;
+				ORGANIZATIONNAME = SwifterSwift;
```

Comment:
This will make sure that whenever a new file is created it won't show `//  Copyright © 2018 ___ORGANIZATIONNAME___`